examples/animation/histogram.py

```diff
@@ -30,12 +28,20 @@
 top = bottom + n
 nrects = len(left)
 
-# here comes the tricky part -- we have to set up the vertex and path
-# codes arrays using moveto, lineto and closepoly
-
-# for each rect: 1 for the MOVETO, 3 for the LINETO, 1 for the
-# CLOSEPOLY; the vert for the closepoly is ignored but we still need
-# it to keep the codes aligned with the vertices
+###############################################################################
+# Here comes the tricky part -- we have to set up the vertex and path codes
+# arrays using `plt.Path.MOVETO`, `plt.Path.LINETO` and `plt.Path.CLOSEPOLY`
```

Comment:
`` double backticks for rST (do this here and in all subsequent uses of backticks)